coverage.txt

```diff
@@ -0,0 +1,682 @@
+mode: atomic
```

Comment:
Should this be in the pr?

Reply:
is there an easy way to remove changes to a file on a commit? gonna look into this

Reply:
It looks like this file was added.  You can simply delete it and commit.  When we squash and merge the changes won't be there.  And the review tooling will not show the changes once you commit a delete.

Reply:
removed